index.html

```diff
@@ -9,16 +9,43 @@
   </head>
   <body>
     <header>
-      <nav class="header-nav-mobile">
+      <div class="header-nav-mobile">
         <a href="/" class="logo">
           <h3>MHR</h3>
         </a>
         <button type="button" class="menu-bar">
           <img src="Imgs/Icon - Menu.svg" alt="" />
         </button>
-      </nav>
+        <nav class="header-nav-mobile-hidden">
+          <button type="button" class="menu-bar-close">
+            <img src="Imgs/Icon-Cancel.svg" alt="cancel icon" />
+          </button>
+          <div class="header-nav-desktop">
+            <ul class="menu-list">
+              <li class="menu-item">
+                <a href="#jobs-id" class="link-portfolio"> Portfolio </a>
+              </li>
+              <li class="menu-item">
+                <a href="#about-myself-id" class="link-about"> About </a>
+              </li>
+              <li class="menu-item">
+                <a href="#footer-container-id" class="link-contact">
+                  Contact
+                </a>
+              </li>
+            </ul>
+          </div>
+        </nav>
+      </div>
```

Comment:
- [ ] Your desktop view looks very good but the mobile view does not look good any longer. Kindly ensure that the your app on mobile still has proper margin, padding, and element in the right position.

![image](https://user-images.githubusercontent.com/84156714/235983734-21cb412d-3122-41c1-b0d5-2ad2cf1e8eea.png)

- [ ] Please go through all the sections and ensure that they look good on mobile view as well. I noticed that your contact section is scattered in mobile view.

![image](https://user-images.githubusercontent.com/84156714/235984392-1d678e8d-c0b0-4746-9c4b-3d9eb5fef1f9.png)